src/js/questionPage.js

```diff
@@ -0,0 +1,66 @@
+class ProgressBar {
+	constructor(seconds) {
+		this.secondsToCount = seconds;
+		this.countDownDate = new Date();
+		this.countDownDate.setSeconds(
+			this.countDownDate.getSeconds() + this.secondsToCount
+		);
+		this.createProgressBar();
+		this.element = document.getElementById("progress-bar-text");
+
+		this.startBar();
+		this._status = true;
+	}
+
+	createProgressBar() {
+		let e = document.createElement("div");
+		e.className = "round-time-bar";
+		e.id = "progress";
+		e.innerHTML =
+			"<div></div><p class='progress-bar-text' id='progress-bar-text'></p>";
+
+		let healthBar = document.querySelector(".health");
+		let main = document.querySelector(".container");
+		main.insertBefore(e, healthBar);
+		document.documentElement.style.setProperty("--duration", 15);
+	}
+
+	startBar() {
+		this.element.innerHTML = this.secondsToCount + "s";
+		const x = setInterval((date = this.countDownDate, elem = this.element) => {
```

Comment:
We should avoid naming variables like x, y, a, b, c etc. Variable name should tell us what is it, in this case it's interval id.